core/src/main/java/io/github/cpaech/charlie/Controller.java

```diff
@@ -21,6 +21,28 @@ public void render(float delta) {
             model.scoreA++; // player A scores a point
             resetBall();    // Ball zurücksetzen
         }
+        // keeps the ball updated based on the delta time and the current velocity
+        // Delta is the time since the last frame, used for smooth movement
+        model.ballVelocity.x = model.ballVelocity.x * delta;
+        model.ballVelocity.y = model.ballVelocity.y * delta;
+
+        // Collision with Paddle A
+        if (model.ball.overlaps(model.paddleA)) {
+            model.ballVelocity.x *= -1; // x-Richtung umkehren
```

Comment:
Sollte -1.0f - Weil float -> evtl rundungsfehler?
```suggestion
            model.ballVelocity.x *= -1.0f; // x-Richtung umkehren
```